hwdb/60-evdev.hwdb

```diff
@@ -146,6 +146,17 @@ evdev:name:Elan Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnUX305UA:*
  EVDEV_ABS_35=0:3097:32
  EVDEV_ABS_36=0:2119:33
 
+#########################################
+# Bangho
+#########################################
+
+# Bangho Cloud Pro
+evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:*svnBANGHO:pnCLOUDPRO:*
+ EVDEV_ABS_00=52:1747:17
+ EVDEV_ABS_01=45:954:14
+ EVDEV_ABS_35=52:1747:17
+ EVDEV_ABS_36=45:954:14
+ 
```

Comment:
there's an extra space in this line here, causing the tests to fail. If you remove that space and force-push to your branch, everything should work.

btw, best to run `ninja -C builddir test` before pushing, saves you the roundtrips. replace `builddir` with whatever you passed to meson.